--- a/Modules/Panels/Settings/Tabs/Display/DisplayTab.qml
+++ b/Modules/Panels/Settings/Tabs/Display/DisplayTab.qml
@@ -72,6 +72,11 @@
       tabIndex: 1
       checked: subTabBar.currentIndex === 1
     }
+    NTabButton {
+      text: I18n.tr("panels.display.scale-tab")
+      tabIndex: 2
+      checked: subTabBar.currentIndex === 2
+    }
   }

   Item {
@@ -88,5 +93,6 @@
     NightLightSubTab {
       timeOptions: timeOptions
       onCheckWlsunset: wlsunsetCheck.running = true
     }
+    ScaleSubTab {}
   }
 }
