--- a/Modules/Panels/Settings/SettingsContent.qml
+++ b/Modules/Panels/Settings/SettingsContent.qml
@@ -24,6 +24,7 @@
 import qs.Modules.Panels.Settings.Tabs.SystemMonitor
 import qs.Modules.Panels.Settings.Tabs.UserInterface
 import qs.Modules.Panels.Settings.Tabs.Wallpaper
+import qs.Modules.Panels.Settings.Tabs.Spotify
 import qs.Services.Compositor
 import qs.Services.Power
 import qs.Services.System
@@ -498,6 +499,10 @@
     id: desktopWidgetsTab
     DesktopWidgetsTab {}
   }
+  Component {
+    id: spotifyTab
+    SpotifyTab {}
+  }

   function updateTabsModel() {
     let newTabs = [
@@ -633,6 +638,12 @@
             "icon": "settings-about",
             "source": aboutTab
           }
+          ,{
+            "id": SettingsPanel.Tab.Spotify,
+            "label": "panels.spotify.title",
+            "icon": "settings-spotify",
+            "source": spotifyTab
+          }
         ];

     root.tabsModel = newTabs;
